sormas-ui/src/main/java/de/symeda/sormas/ui/dashboard/campaigns/CampaignDashboardDiagramComponent.java

```diff
@@ -239,6 +241,20 @@ public void buildDiagramChart(String title, boolean isCommunityGrouping) {
 		hcjs.append("]");
 		hcjs.append("}");
 		campaignColumnChart.setHcjs(hcjs.toString());
+		if (showPercentages) {
```

Comment:
This now displays the warning below every diagram that shows percentages, even if they don't use population data at all or there is no missing population data. Also, the hint stays on the screen even if the user changes from population data to total values. It should always be hidden in that case.